.gitignore

```diff
@@ -16,4 +16,7 @@ vendor/*
 vendor/bundle/*
 vendor/cache/*
 vendor/plugins/*
-coverage/*
\ No newline at end of file
+coverage/*
```

Comment:
I might be wrong but this says it's okay to have an empty coverage folder. Why not have `coverage`